thirdeye/thirdeye-frontend/app/utils/yaml-tools.js

```diff
@@ -12,12 +12,11 @@ dataset: dataset_to_which_this_metric_belongs
 rules:
 - detection:
   - name: detection_rule_1
-    type: ALGORITHM             # Configure the detection type here. See doc for more details.
+    type: PERCENTAGE_RULE       # Configure the detection type here. See doc for more details.
```

Comment:
+ @harleyjj 
Can we customize and configure the default template? At LinkedIn, we are relying on ALGORITHM as the default detection. If we change this, we need to have an alternative to plug a different default setting.

Reply:
Hey @akshayrai 

`ALGORITHM` doesn't seem to work in open source. It says type not found. I changed it so that we can have a default that users can use out of the box.

Reply:
I think we can move these templates into the environment.js file [here](https://github.com/apache/incubator-pinot/blob/master/thirdeye/thirdeye-frontend/config/environment.js) and then different users can configure their own default templates for detection and subscription group yaml configurations.

Reply:
@harleyjj @akshayrai 
We could keep the default setting for now if it helps and come up with a plan. I did this now since I thought that have a working default setting would be easier for a user to get started.

Reply:
Yes, we should pull this out into a config. @suvodeep-pyne, feel free to update this or we can do it at a later stage.